6_predict/src/plot_diagnostics.R

```diff
@@ -447,6 +447,101 @@ make_class_prediction_map <- function(class_probs, reaches, out_dir,
   return(fnames)
 }
 
+
+make_transition_region_map <- function(class_probs, reaches,
+                                       region_threshold,
+                                       out_dir, model_name, pt_size = 0.5) {
+  #' @description this function creates a map showing the transition regions 
+  #' (reaches not distinctly predicted in single cluster)
+  #' 
+  #' @param class_probs dataframe of predicted class probabilities for each reach.
+  #' must have an "ID" column and columns for the class probabilities labeled with
+  #' the name of the class. No other columns. 
+  #' @param reaches sf object containing the reaches to plot. Must have a "COMID" column
```

Comment:
COMID is set to ID in the function call. 
```suggestion
  #' @param reaches sf object containing the reaches to plot. Must have an "ID" column corresponding to the same "ID" values in class_probs
```